bedrock/settings/base.py

```diff
@@ -1739,6 +1742,9 @@ def JINJA_CONFIG():
         'source_filenames': (
             'js/libs/modernizr.custom.shiv-load.js',
             'js/base/mozilla-input-placeholder.js',
+            'js/libs/jquery.waypoints.min.js',
```

Comment:
I do share your concern though - the new uglifier seems more picky than the old one, so we are right to be more cautious now.